lib/modules/tokens/TokensProvider.tsx

```diff
@@ -31,24 +31,27 @@ export function _useTokens(
   variables: GetTokensQueryVariables
 ) {
   const skipQuery = useSkipInitialQuery(variables)
+  const pollInterval = mins(3).toMs()
 
   // skip initial fetch on mount so that initialData is used
   const { data: tokensData } = useQuery(GetTokensDocument, {
     variables,
     skip: skipQuery,
   })
-  const { data: tokenPricesData, loading: isLoadingTokenPrices } = useQuery(
-    GetTokenPricesDocument,
-    {
-      variables,
-      // The server provides us with an initial data set, but we immediately reload the potentially
-      // stale data to ensure the prices we show are up to date. Every 3 mins, we requery token prices
-      initialFetchPolicy: 'no-cache',
-      nextFetchPolicy: 'cache-and-network',
-      pollInterval: mins(3).toMs(),
-      notifyOnNetworkStatusChange: true,
-    }
-  )
+  const {
+    data: tokenPricesData,
+    loading: isLoadingTokenPrices,
+    startPolling: startTokenPricePolling,
+    stopPolling: stopTokenPricePolling,
+  } = useQuery(GetTokenPricesDocument, {
```

Comment:
Lets not export the polling interval as well since it's internal context. We can do this instead:
```suggestion
    startPolling,
    stopPolling,
  } = useQuery(GetTokenPricesDocument, {

const startTokenPricePolling = () => startPolling(pollInterval)
const stopTokenPricePolling = () => stopPolling(pollInterval)
```
And export those.

Reply:
Updated this.